src/WebJobs.Script.WebHost/web.InProcess.win-x64.xdt

```diff
@@ -4,6 +4,6 @@
     <handlers>
       <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModuleV2" resourceType="Unspecified" xdt:Transform="Replace" xdt:Locator="Match(name)" />
     </handlers>
-    <aspNetCore processPath=".\Microsoft.Azure.WebJobs.Script.WebHost.exe" arguments="" stdoutLogEnabled="false" hostingModel="inprocess" xdt:Transform="Replace" />
+    <aspNetCore processPath="D:\Program Files\dotnet\dotnet.exe" arguments=".\Microsoft.Azure.WebJobs.Script.WebHost.dll" stdoutLogEnabled="false" hostingModel="inprocess" xdt:Transform="Replace" />
```

Comment:
Won't the 64-bit flavor be a self-contained app instead? That would need the process name, correct?

Reply:
New App service OS(ANT79) already includes ASP.NET Core 2.2 x64 so we do not need self-contained app for x64 anymore